.travis.yml

```diff
@@ -14,7 +14,6 @@ env:
     - CONDA_NPY=111  CONDA_PY=35
     - CONDA_NPY=112  CONDA_PY=35
     - CONDA_NPY=111  CONDA_PY=36
-    - CONDA_NPY=112  CONDA_PY=36
```

Comment:
Unless you are in a hurry don't do that. I will hunt the dependencies down and get them build for Python 3.6.